src/lib/ngc/compile-source-files.ts

```diff
@@ -56,5 +56,5 @@ export async function compileSourceFiles(
   }
 
   const exitCode = ng.exitCodeFromResult(result.diagnostics);
-  return exitCode === 0 ? Promise.resolve() : Promise.reject(new Error(ng.formatDiagnostics(result.diagnostics)));
+  return exitCode === 0 ? Promise.resolve() : Promise.reject(ng.formatDiagnostics(result.diagnostics));
```

Comment:
Note: no need to re-wrap this in an error, as like this, the error message will be be displayed twice.